enhancements/ingress/haproxy-thread-tuning.md

```diff
@@ -0,0 +1,216 @@
+---
+title: haproxy-thread-tuning
+authors:
+  - "@rfredette"
+reviewers:
+  - "@Miciah"
+  - "@danehans"
+  - "@frobware"
+  - "@sgreene570"
+  - "@knobunc"
+  - "@miheer"
+  - "@candita"
+approvers:
+  - "@knobunc"
+  - "@miciah"
+  - "@frobware"
+  - "@danehans"
+creation-date: 2021-03-24
+last-updated: 2021-03-26
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced|informational
```

Comment:
```suggestion
status: implementable
```
...But let's get this in a follow-up, when we can change the status to "implemented".  * grin *.